lib/test.py

```diff
@@ -233,6 +234,9 @@ def run(self, server):
                 is_tap, is_ok, is_skip = self.check_tap_output()
                 self.is_equal_result = is_ok
                 self.skip = is_skip
+
+            server.stop(silent=True)
```

Comment:
It seems, this patch is not enough to fail on the scenario from #416, because `SIGABRT` is considered as OK (it was added as part of the #398 fix).